HW02_B_Mimic_starter - Viola graphs.Rmd

```diff
@@ -0,0 +1,178 @@
+#or you can set the theme for all the graphs using theme_set()
+#theme_bw() is best theme (IMO)
+
+#for graph 3:
+library("ggrepel")
+
+
+```
+
+## HW02 Part B
+
+For this part of the HW, the goal is to try to recreate the graphs I make from scratch. I will only provide the MD, not the actual code I used to create it besides which data I use to create it. The rest will be up to you. 
+
+Try for all 4, but if you are struggling don't worry about it. Try your best for each, if you don't get everything that's what the peer-review is for. :smile:
+
+
+### Graph 1
+```{r, echo = T, include = TRUE}
+
+#hint think about the *position* the bars are in...
+```
+
+
+Using the diamonds dataset, make this graph:
+```{r graph1 code, echo=FALSE}
+
+data("diamonds")
+
+diamonds %>% 
+  ggplot(mapping = aes(x = cut, fill = clarity)) +
+  geom_bar(position = position_dodge(width = 0.9)) +
+  labs(
+      title = "My Diamond Collection",
+      subtitle = "Boxplot representing the number of diamonds in my diamond collection by \ntype of cut quality and clarity of diamond",
+      x = "Diamond Cut",
+      y = "Number of Diamonds"
+  ) +
+  theme(plot.title = element_text(hjust = 0.5)) +
+  annotate("text", x = "Premium", y = 4500, label = "My Best Diamonds, \n of course") +
+  annotate ("rect", xmin = 4.5, xmax = 5.5, ymin = 0, ymax = 5000, hjust = "Ideal", fill = "grey40", alpha = 0.3)
+
+```
+
+### Graph 2
+```{r, echo = T, include = TRUE}
+data("iris")
+
+```
+
+Using the iris dataset, make this graph:
+```{r graph 2 code, echo=FALSE}
+data("iris")
+
+arrange(iris, Species)
+
+iris %>% 
+  ggplot(mapping = aes(x = Sepal.Length, y = Petal.Length, color = Species)) +
+  geom_point() +
+  geom_smooth(method= lm, se = F, color = "black") +
+  facet_wrap(~ Species, scales = "free_y")
+
+```
+
+
+### Graph 3
+You'll need the information in this first box to create the graph
+```{r, echo = TRUE}
+data("mpg")
+corvette <- mpg[mpg$model == "corvette",]
+#install
+require("ggrepel") #useful for making text annotations better, hint hint
+set.seed(42)
+```
+
+Now using the mpg dataset and the corvette dataset, make this graph:
+
+```{r graoh 3 code}
+
+#Defining a new data frame that includes only the data for corvettes, and has "corvette" replaced with "Corvette".
+
+a <- replace((mpg %>% filter(model=="corvette")), (mpg %>% filter(model=="corvette")) == "corvette", "Corvette")
+
+ggplot(mpg, aes(x=displ, y=hwy)) +
+  geom_point() +
+  
+  geom_point(data=(mpg %>% filter(model=="corvette")), aes(x=displ, y=hwy), color = "blue") +
+  xlim(1,8) +
+  
+  #Initially tried to use: geom_text(data=(a %>% filter(model=="Corvette")), aes(label=paste(model, "," , year)), color = "black", nudge_x=0.2, nudge_y=-2), but that would only let me move the labels by the same amount in x and y directions, so I decided to switch to annotate, since there were only 5 labels to add (not that much work really).
+  
+  annotate("text", x = 6.35, y = 22, label = "Corvette, 1999") +
```

Comment:
Oh wow you did this all by hand, that is commitment! I think you could have saved yourself some time with the "paste" function, which would bring those two factors together as one graphable datapoint